assets/js/components/resend_invite.ts

```diff
@@ -7,7 +7,8 @@ $(() => {
 
   if (!!flashUl && !!resendInviteLinks) {
     resendInviteLinks.forEach(item => {
-      item.addEventListener('mousedown', (event) => {
+      item.addEventListener('click', (event) => {
+        event.preventDefault();
         event.stopImmediatePropagation();
```

Comment:
Isnt it strange that this solves the issue?

Mousedown should only be invoked once on each click?

Reply:
discussed, approved